CONTRIBUTING.md

```diff
@@ -128,32 +131,17 @@ Meilisearch tools follow the [Semantic Versioning Convention](https://semver.org
 This project integrates a bot that helps us manage pull requests merging.<br>
 _[Read more about this](https://github.com/meilisearch/integration-guides/blob/main/resources/bors.md)._
 
-### Automated Changelogs <!-- omit in TOC -->
-
-This project integrates a tool to create automated changelogs.<br>
-_[Read more about this](https://github.com/meilisearch/integration-guides/blob/main/resources/release-drafter.md)._
+### Release on Github and npm
 
-### How to Publish the Release <!-- omit in TOC -->
+This repository uses the [changesets](https://github.com/Noviny/changesets) library to handle the version updates and the publishing on Github and `npm`.
 
-⚠️ Before doing anything, make sure you got through the guide about [Releasing an Integration](https://github.com/meilisearch/integration-guides/blob/main/resources/integration-release.md).
+When PR's are merged on `main`, they trigger the [`release` CI](./.github/workflows/publish.yml). This CI creates a PR titled `Version Packages` containing all the version updates and changelogs of the impacted packages. The PR updates the versions and the changelogs based on the `changesets` that were previously pushed on main (see [changeset section](#changesets)).
 
-#### Version update
-Make a PR modifying the following files with the right version:
+To release on Github and `npm` you must merge the `Version packages` PR. This will trigger the publishing action and create github releases and `npm` releases for all affected packages.
```

Comment:
```suggestion
To release on Github and `npm` you must merge the `Version packages` PR. This will trigger the publishing action and create the github and `npm` releases for all affected packages.
```